File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.py
import sys

T = int(sys.stdin.readline())

coin = [25, 10, 5, 1]

for i in range(T) :
    N = int(sys.stdin.readline())

    coinCount = []

    for j in coin :
        coinCount.append(N//j)
        N = N % j
    print(*coinCount)

File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringBuilder sb = new StringBuilder();
		StringTokenizer st;
		
		st = new StringTokenizer(br.readLine());
		int N = Integer.parseInt(st.nextToken());
		
		int[] coin = {25,10,5,1};
		
		for (int i = 0; i < N; i++) {
			st = new StringTokenizer(br.readLine());
			int num = Integer.parseInt(st.nextToken());
			
			for (int j : coin) {
				sb.append(num/j + " ");
				num%=j;
			}
			sb.append("\n");
		}
		System.out.println(sb);
		br.close();
	}
}